client/dashboard/sites/domain-management/placeholder.tsx

```diff
@@ -0,0 +1,9 @@
+import { useParams } from '@tanstack/react-router';
+import { PageHeader } from '../../components/page-header';
+import PageLayout from '../../components/page-layout';
+
+export default function DomainManagementPlaceholder() {
+	const { domainName } = useParams( { from: '/sites/$siteSlug/domains/$domainName' } );
```

Comment:
We can import `siteDomainManagementRoute` and use `siteDomainManagementRoute.useParams()` to get parameters 🙂